src/components/tables/AccessGrants/index.tsx

```diff
@@ -28,8 +28,12 @@ function AccessGrantsTable() {
                     query,
                     ['user_full_name', 'subject_role', 'object_role'],
                     searchQuery,
-                    columnToSort,
-                    sortDirection,
+                    [
+                        {
+                            col: columnToSort,
+                            direction: sortDirection,
+                        },
+                    ],
```

Comment:
We now always pass an array of sorting options so this change happened in a lot of files.